mesos-swarm-marathon/configure-mesos-cluster.sh

```diff
@@ -11,6 +11,7 @@
 ###########################################################
 
 echo "starting mesos cluster configuration"
```

Comment:
Your script does not have set -x or set -e, that's not exactly hardening :-) 

If one of these lines fail, script will keep going. Is that what you intended @anhowe ?